source/normalize-arguments.ts

```diff
@@ -431,14 +433,6 @@ export const normalizeRequestArguments = async (options: NormalizedOptions): Pro
 		options.lookup = options.dnsCache.lookup;
 	}
 
-	/* istanbul ignore next: electron.net is broken */
-	// No point in typing process.versions correctly, as
-	// `process.version.electron` is used only once, right here.
-	if (options.useElectronNet && (process.versions as any).electron) {
-		const electron = dynamicRequire(module, 'electron') as any; // Trick webpack
-		options.request = electron.net.request ?? electron.remote.net.request;
-	}
```

Comment:
I think we should just leave this. No other Electron-related code changes. That's the least risky way to handle it. Just add a deprecation message here.

Reply:
Do you mean leave this as is or move code from `electron-request.ts` to `request-as-event-emitter.ts`? Personally I prefer to store it in a separate file so it doesn't clutter the Got core.

If you mean the first, then no, it won't work as Electron needs fixing to support POST etc.

Reply:
> If you mean the first, then no, it won't work as Electron needs fixing to support POST etc.

My argument was that we don't add any Electron fixes. Electron will work (or rather not work) like it currently does. It's much easier to deprecate it if it's not even working.

Reply:
I see. Will do :D